libs/blocks/merch-card/merch-card.js

```diff
@@ -210,36 +210,43 @@ const parseContent = async (el, merchCard) => {
       }
       return;
     }
-    if (tagName === 'H6' && element.firstElementChild?.tagName === 'EM') {
-      const calloutContentWrapper = createTag('div');
-      const calloutContent = createTag('div');
-      const emElement = element.firstElementChild;
-      let imgElement = null;
-      const fragment = document.createDocumentFragment();
-
-      emElement.childNodes.forEach((child) => {
-        if (child.nodeType === Node.ELEMENT_NODE && child.tagName === 'A' && child.innerText.trim().toLowerCase() === '#icon') {
-          const [imgSrc, tooltipText] = child.getAttribute('href')?.split('#') || [];
-          imgElement = createTag('img', {
-            src: imgSrc,
-            title: decodeURIComponent(tooltipText),
-            class: 'callout-icon',
-          });
-        } else {
-          const clone = child.cloneNode(true);
-          fragment.appendChild(clone);
-        }
-      });
+    if (tagName === 'H6') {
+      if (element.firstChild.nodeType === Node.TEXT_NODE) {
+        const paymentDetails = createTag('div', { slot: 'payment-details' });
+        paymentDetails.innerHTML = element.innerHTML;
```

Comment:
This might trigger some Kodiak issues, maybe a spread or recursive append would be better suited. This could also be troublesome if some event listeners have already been added to `element`. Also, could `element` be used after this piece of logic? If not, maybe it could be deleted.